README.rst

```diff
@@ -35,10 +35,11 @@ Open house
 
 We regularly meet in an open-for-all Google hangout to discuss progress and plans for Cartographer.
 
-The next Cartographer Open House Hangout is on Thursday, October 12, 5pm CEST (8am PST) [`Hangouts link`_].
+The next Cartographer Open House Hangout is on Thursday, October 26, 5pm CEST (8am PST) [`Hangouts link`_].
 
 .. _Hangouts link: https://staging.talkgadget.google.com/hangouts/_/google.com/cartographeropenhouse
 
+- October 12, 2017: `Slides <https://storage.googleapis.com/cartographer-public-data/cartographer-open-house/171012/slides.pdf`_
```

Comment:
Typo, missing >. Doublecheck by using the View button on github.

Reply:
Done